tools/sync-labels/config.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "repositories": [
+      "This list gets automatically populated when running the generate-repository-list script"
+    ],
+    "milestones": [],
+    "labels": [
+      {
+        "name": "difficulty:easy",
+        "color": "bfe5bf"
+      },
+      {
+        "name": "difficulty:moderate",
+        "color": "bfe5bf"
+      },
+      {
+        "name": "difficulty:hard",
+        "color": "bfe5bf"
```

Comment:
all difficulties have the same color, that seems wrong to me

Reply:
I'm unsure, but think it was like that in ipfs/js-ipfs and where these labels are copied from. But, we can change it of course, to something that doesn't collide with the color scale priority-labels use.

Reply:
going to merge as it is what it is for now :)